pages.fr/common/vagrant-plugin.md

```diff
@@ -0,0 +1,29 @@
+# vagrant plugin
+
+> Gère les plugiciels Vagrant.
+> Voir aussi : `vagrant`.
+> Plus d’informations : <https://developer.hashicorp.com/vagrant/docs/cli/plugin>.
```

Comment:
```suggestion
> Plus d'informations : <https://developer.hashicorp.com/vagrant/docs/cli/plugin>.
```

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md#fr

This will solve this bot check:

> The page pages.fr/common/vagrant-plugin.md has a more info link that does not match the one in the English page. Please check the "More information:" translation as well using https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md.

The link is already the same.